recipes/samurai/bld.bat

```diff
@@ -0,0 +1,5 @@
+cmake ${CMAKE_ARGS}  -G "Ninja" -D CMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX% %SRC_DIR% -B build -D CMAKE_SYSTEM_PROCESSOR=x64
```

Comment:
Does the `${CMAKE_ARGS}` syntax really work here? I would think that only `%CMAKE_ARGS%` works in `cmd.exe`.